qualtran/bloqs/msft_resource_estimator_interop.ipynb

```diff
@@ -0,0 +1,136 @@
 T factories can be utilized during the algorithm's runtime\",\n       'explanation': 'This is the maximum number of T factories used for producing the demanded T states, which can be created and executed by the algorithm in parallel.'},\n      {'path': 'physicalCountsFormatted/maxDuration',\n       'label': 'Maximum runtime duration',\n       'description': 'The maximum runtime duration allowed for the algorithm runtime',\n       'explanation': 'This is the maximum time allowed to the algorithm. If specified, the estimator targets to minimize the number of physical qubits consumed by the algorithm for runtimes under the maximum allowed.'},\n      {'path': 'physicalCountsFormatted/maxPhysicalQubits',\n       'label': 'Maximum number of physical qubits',\n       'description': 'The maximum number of physical qubits allowed for utilization to the algorith',\n       'explanation': 'This is the maximum number of physical qubits available to the algorithm. If specified, the estimator targets to minimize the runtime of the algorithm with number of physical qubits consumed not exceeding this maximum.'}]}],\n   'assumptions': ['_More details on the following lists of assumptions can be found in the paper [Accessing requirements for scaling quantum computers and their applications](https://aka.ms/AQ/RE/Paper)._',\n    '**Uniform independent physical noise.** We assume that the noise on physical qubits and physical qubit operations is the standard circuit noise model. In particular we assume error events at different space-time locations are independent and that error rates are uniform across the system in time and space.',\n    '**Efficient classical computation.** We assume that classical overhead (compilation, control, feedback, readout, decoding, etc.) does not dominate the overall cost of implementing the full quantum algorithm.',\n    '**Extraction circuits for planar quantum ISA.** We assume that stabilizer extraction circuits with similar depth and error correction performance to those for standard surface and Hastings-Haah code patches can be constructed to implement all operations of the planar quantum ISA (instruction set architecture).',\n    '**Uniform independent logical noise.** We assume that the error rate of a logical operation is approximately equal to its space-time volume (the number of tiles multiplied by the number of logical time steps) multiplied by the error rate of a logical qubit in a standard one-tile patch in one logical time step.',\n    '**Negligible Clifford costs for synthesis.** We assume that the space overhead for synthesis and space and time overhead for transport of magic states within magic state factories and to synthesis qubits are all negligible.',\n    '**Smooth magic state consumption rate.** We assume that the rate of T state consumption throughout the compiled algorithm is almost constant, or can be made almost constant without significantly increasing the number of logical time steps for the algorithm.']}}}",
```

Comment:
I remember there was also a way to plot a UI, correct? Can you also add the example to do that?

Reply:
![Screenshot 2024-03-06 at 1 39 30 PM](https://github.com/quantumlib/Qualtran/assets/28879561/7170f62e-f0ed-4a8b-be62-7f505f32289a)

@tanujkhattar when this code runs it shows this at the end. Is this what you were referring to? This is also what I had in the QFT demonstration.

Reply:
I remember seeing plots in the Microsoft's demonstration? For example https://learn.microsoft.com/en-us/azure/quantum/overview-resource-estimator-output-data#space-time-diagram using `qsharp_widgets`